packages/@aws-cdk/cdk/lib/tag-aspect.ts

```diff
@@ -104,7 +109,7 @@ export class Tag extends TagBase {
   }
 
   protected applyTag(resource: ITaggable) {
-    if (resource.tags.applyTagAspectHere(this.props.includeResourceTypes, this.props.excludeResourceTypes)) {
+    if (resource.tags.applyTagAspectHere(this.props.includeResourceTypes, this.props.excludeResourceTypes) || Stack.isStack(resource)) {
```

Comment:
Actually looking at this again, do you need `|| Stack.isStack(resource)` ? I think if you remove that everything should still work or am I missing something?

Reply:
@moofish32 , Just commit a change with this. 

I had to move the isTaggable up to the Contructor as, otherwise, I had to duplicate it in the "Stack" area.